src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -115,11 +149,43 @@ export class AssetsTable extends React.Component {
     );
   }
 
+  renderStatusAlert() {
+    const { assetsStatus } = this.props;
+    const deleteAssetName = this.state.assetToDelete.display_name;
+    let alertDialog = `Deleting ${deleteAssetName}`;
+    let alertType = 'info';
+
+    if (assetsStatus.text === AssetsConstants.ERROR_FILE_DELETE) {
+      alertDialog = `Unable to delete ${deleteAssetName}.`;
```

Comment:
Not something to worry about now, but we'll need for this to be internationalized too (e.g. string filled in via template) or some way to do this. CC: @efischer19

Reply:
👍 Agreed. We'll also need it for the Modal body text & buttons.